transformations/sentence_additions/requirements.txt

```diff
@@ -0,0 +1 @@
+transformers==4.7.0
```

Comment:
The main requirements.txt already contains a transformations version.

Reply:
I'll remove this. Thanks :)